dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/uploadconfig/FrameworkUploadConfigBuilder.kt

```diff
@@ -0,0 +1,71 @@
+package org.dataland.frameworktoolbox.specific.uploadconfig
+
+import org.dataland.frameworktoolbox.intermediate.Framework
+import org.dataland.frameworktoolbox.specific.uploadconfig.elements.SectionUploadConfigBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkBooleanLambda
+import org.dataland.frameworktoolbox.utils.DatalandRepository
+import org.dataland.frameworktoolbox.utils.LoggerDelegate
+import org.dataland.frameworktoolbox.utils.capitalizeEn
+import org.dataland.frameworktoolbox.utils.freemarker.FreeMarker
+import java.io.FileWriter
+import java.nio.file.Path
+import kotlin.io.path.div
+
+/**
+ * A FrameworkUploadConfigBuilder converts an Intermediate-Representation framework to a TypeScript Input-Configuration
+ * and integrates the generated code into a Dataland Repository.
+ * @param framework the framework DataModel to convert
+ */
+class FrameworkUploadConfigBuilder(
+    private val framework: Framework,
+) {
+    companion object {
+//        private const val ESLINT_TIMEOUT = 60L
+    }
+
+    private val logger by LoggerDelegate()
+
+    val rootSectionConfigBuilder = SectionUploadConfigBuilder(
+        parentSection = null,
+        label = "root-section",
+        expandOnPageLoad = false,
+        shouldDisplay = FrameworkBooleanLambda.TRUE,
+        name = "???",
```

Comment:
Ah. Yes. The `???` ;)

Reply:
😆  @AdrianBeyer  can you take a look at this?

Reply:
Fixed